typespec-tests/src/main/java/com/parameters/bodyoptionality/BodyOptionalityClient.java

```diff
@@ -116,9 +118,11 @@ public void requiredExplicit(BodyModel body) {
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public void requiredImplicit(BodyModel bodyModel) {
+    public void requiredImplicit(String name) {
```

Comment:
This is expected after behavior change.

Test is "requiredImplicit", so I didn't add `@body` to this case in cadl-ranch.